tests/contrib/django_util/settings.py

```diff
@@ -12,8 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This module provides a base Django settings module used by the rest of the
+tests."""
+
 import os
 
+INSTALLED_APPS = [
```

Comment:
It's a Django configuration, needs to be a list: https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-INSTALLED_APPS